dashboard/db/schema.rb

```diff
@@ -742,6 +742,8 @@
     t.integer  "script_id",       limit: 4
     t.integer  "level_source_id", limit: 4
     t.boolean  "submitted"
+    t.boolean  "view_answers"
```

Comment:
The field means that the user_level is in a state where the student can view answers, but can't actually edit. I was originally going to call it readonly, but (a) we already have the readonly concept in a bunch of other places and (b) though this is readonly, it's really more than that, since the important bit is that users can view answers.

Thoughts on a better column name that would make this more clear?